src/main/java/com/commercetools/sync/inventories/utils/InventorySyncUtils.java

```diff
@@ -0,0 +1,82 @@
+package com.commercetools.sync.inventories.utils;
+
+import com.commercetools.sync.commons.BaseSyncOptions;
+import com.commercetools.sync.inventories.InventorySyncOptions;
+import com.commercetools.sync.services.TypeService;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.commons.utils.CustomUpdateActionUtils.buildCustomUpdateActions;
+import static com.commercetools.sync.inventories.utils.InventoryDraftTransformerUtils.transformToDraft;
+import static com.commercetools.sync.inventories.utils.InventoryUpdateActionUtils.buildChangeQuantityAction;
+import static com.commercetools.sync.inventories.utils.InventoryUpdateActionUtils.buildSetExpectedDeliveryAction;
+import static com.commercetools.sync.inventories.utils.InventoryUpdateActionUtils.buildSetRestockableInDaysAction;
+import static com.commercetools.sync.inventories.utils.InventoryUpdateActionUtils.buildSetSupplyChannelAction;
+import static java.util.stream.Collectors.toList;
+
+/**
```

Comment:
to me it looks like this class provides factory methods for assembling update actions rather than
actual synchronising entries